DevEdu.API/DevEdu.DAL/Repositories/StudentAnswerOnTaskRepository.cs

```diff
@@ -175,5 +176,50 @@ public List<StudentAnswerOnTaskForTaskDto> GetStudentAnswersToTaskByTaskId(int i
                     commandType: CommandType.StoredProcedure)
                 .ToList();
         }
+
+        public List<StudentAnswerOnTaskDto> GetStudentAnswersOnTasksByGroupId(int groupId, int userId, List<GroupTaskDto> groupTaskDto)
```

Comment:
please change the implementation of this method (and rename it) according to comments from above